src/app/test-record/defects/defects.component.html

```diff
@@ -0,0 +1,75 @@
+
+  <dl *ngFor="let defect of testType.defects" class="govuk-summary-list">
+    <h2 class="govuk-heading-l">
+      <span class="govuk-!-margin-top-3"> {{defect.deficiencyRef}}</span> -
+      <span class="govuk-tag govuk-!-font-size-24"
+            [class.tag--black]="defect.deficiencyCategory==='dangerous'"
+            [class.tag--red]="defect.deficiencyCategory==='major'">
+          {{defect.deficiencyCategory}}
+        </span>
+    </h2>
+    <div class="govuk-summary-list__row">
+      <dt class="govuk-summary-list__key">
+        Category
+      </dt>
+      <dd class="govuk-summary-list__value">
+        {{ ((defect.imNumber ? defect.imNumber : '') + ' ' + (defect.imDescription ? defect.imDescription : '')) | DefaultNullOrEmpty}}
+      </dd>
+    </div>
+    <div class="govuk-summary-list__row">
+      <dt class="govuk-summary-list__key">
+        Item
+      </dt>
+      <dd class="govuk-summary-list__value">
+        {{ ((defect.itemNumber ? defect.itemNumber : '') + ' ' + (defect.itemDescription ? defect.itemDescription : '')) | DefaultNullOrEmpty}}
+      </dd>
+    </div>
+    <div class="govuk-summary-list__row">
+      <dt class="govuk-summary-list__key">
+        Description
+      </dt>
+      <dd class="govuk-summary-list__value">
+        {{ ((defect.deficiencyId ? defect.deficiencyId : '') + ' ' + (defect.deficiencySubId ? defect.deficiencySubId : '') + ' ' + (defect.deficiencyText ? defect.deficiencyText : '')) | DefaultNullOrEmpty}}
+      </dd>
+    </div>
+    <div class="govuk-summary-list__row">
+      <dt class="govuk-summary-list__key">
+        Location
+      </dt>
+      <dd class="govuk-summary-list__value">
```

Comment:
is it possible to move this logic to the component side?

Reply:
as it's not in the scope of this ticket better do it when solving bugs from 10283 (test record view ticket) so it can be tested properly